migrations/1665144863994_seed-db-with-predefined-tags.js

```diff
@@ -0,0 +1,37 @@
+
+exports.shorthands = undefined
+
+exports.up = (pgm) => {
+	pgm.sql(`INSERT into tags
```

Comment:
we can use one insert statement for all of them. kindly check out how to use one insert statement to enter multiple values

Reply:
> we can use one insert statement for all of them. kindly check out how to use one insert statement to enter multiple values

i couldn't find anything on that but I will keep searching

Reply:
> https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-insert-multiple-rows/

Done, thanks a lot